praw/models/reddit/subreddit.py

```diff
@@ -410,7 +410,7 @@ def __init__(self, reddit, display_name=None, _data=None):
     def _info_path(self):
         return API_PATH['subreddit_about'].format(subreddit=self)
 
-    def _submit_media(self, data):
+    def _submit_media(self, data, timeout=None):
```

Comment:
Can we make this a required argument since this is an internal function, and we always pass it in?